<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="USER-20161225UH" name="登陆页面测试" tests="5" failures="3" timestamp="6 Jan 2018 13:26:57 GMT" time="44.05" errors="0">
  <testcase name="loginsucces" time="20.127" classname="com.msyidai.cases.login"/>
  <testcase name="loginpasswordflase" time="8.53" classname="com.msyidai.cases.login"/>
  <testcase name="loginusenameflase" time="5.193" classname="com.msyidai.cases.login">
    <failure type="java.lang.AssertionError" message="账户错误 expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: 账户错误 expected [true] but found [false]
	at com.msyidai.cases.login.loginusenameflase(login.java:36)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- loginusenameflase -->
  <testcase name="loginusenameflase" time="4.219" classname="com.msyidai.cases.login">
    <failure type="java.lang.AssertionError" message="账户错误 expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: 账户错误 expected [true] but found [false]
	at com.msyidai.cases.login.loginusenameflase(login.java:36)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- loginusenameflase -->
  <testcase name="loginusenameflase" time="4.553" classname="com.msyidai.cases.login">
    <failure type="java.lang.AssertionError" message="账户错误 expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: 账户错误 expected [true] but found [false]
	at com.msyidai.cases.login.loginusenameflase(login.java:36)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- loginusenameflase -->
</testsuite> <!-- 登陆页面测试 -->
